pandas/io/excel.py

```diff
@@ -191,12 +193,12 @@ def get_writer(engine_name):
 
 
 @Appender(_read_excel_doc)
-def read_excel(io, sheet_name=0, header=0, skiprows=None, skip_footer=0,
-               index_col=None, names=None, parse_cols=None, parse_dates=False,
-               date_parser=None, na_values=None, thousands=None,
-               convert_float=True, converters=None, dtype=None,
-               true_values=None, false_values=None, engine=None,
-               squeeze=False, **kwds):
+def read_excel(io, sheet_name=0, header=0, skiprows=None, nrows=None,
+               skip_footer=0, index_col=None, names=None, parse_cols=None,
```

Comment:
Sure, not a problem. I was using read_csv as a guide and saw that ```nrows``` was directly after ```skiprows```, hence the change.

I will go ahead and line up the read_excel kwargs as close as possible to the read_csv kwargs. I see 3or4 out of order with just a quick glance.